src/providers/fault-count/cat-d/fault-count.cat-d.ts

```diff
@@ -232,7 +244,7 @@ export class FaultCountDHelper {
 
     const seriousFaultSumOfSimpleCompetencies = Object.keys(pickBy(seriousFaults)).length;
     const vehicleCheckSeriousFaults =
-      vehicleChecks ? FaultCountDHelper.getVehicleChecksFaultCountCatD(vehicleChecks).seriousFaults : 0;
+      vehicleChecks ? FaultCountDHelper.getVehicleChecksFaultCountTrailer(vehicleChecks).seriousFaults : 0;
```

Comment:
should be:

`this.getVehicleChecksFaultCountTrailer(vehicleChecks).seriousFaults`